ddtrace/monkey.py

```diff
@@ -46,6 +46,7 @@
     'vertica': True,
     'molten': True,
     'jinja2': True,
+    'mako': False,
```

Comment:
```suggestion
    'mako': True,
```

Reply:
We can enable this by default, that is what we'd like to do for all integrations :)